account-history/performance/index.html

```diff
@@ -0,0 +1,136 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="ROBOTS" content="INDEX, FOLLOW" />
+  <meta name="VIEWPORT" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="https://saxobank.github.io/openapi-samples-js/assets/css/style.css">
+  <script type="text/javascript" src="../../assets/js/boilerplate.js" defer></script>
+  <script type="text/javascript" src="demo.js" defer></script>
+  <title>Demonstration of the performance endpoints</title>
+</head>
+
+<body>
+  <section class="page-header">
+    <h1 class="project-name">Performance</h1>
+    <h2 class="project-tagline">Performance</h2>
+    <a href="https://saxobank.github.io/openapi-samples-js" class="btn">← Back to main page</a>
+    <a href="https://github.com/SaxoBank/openapi-samples-js/account-history/performance" class="btn">View on
+      GitHub</a>
+  </section>
+  <section class="main-content" style="max-width: 68rem !important">
+    <div class="highlight">
+      <label><span>Add token <a href="https://www.developer.saxo/openapi/token" target="_blank"
+            id="idHrefRetrieveToken">from here</a> to the box below:</span><br />
+        <input type="text" class="token-field" id="idBearerToken" value="" placeholder="Paste token here.."
+          autofocus /></label>
+      <input type="button" class="action-button" value="Validate" id="idBtnValidate" /><br />
+      <label>
+        Account: <select class="selector" id="idCbxAccount" <option value="-">Click Validate to populate</option>
+        </select>
+      </label>
+    </div>
+    <h1>Unsettled Amounts</h1>
```

Comment:
Shouldn't this be 'Performance', or 'Portfolio performance'?

Reply:
Ah, you name it "Trade Performance".

Reply:
yup, missed it